src/test/java/jenkins/plugins/git/AbstractGitSCMSourceTest.java

```diff
@@ -503,8 +503,8 @@ public void testSpecificRevisionBuildChooser() throws Exception {
                 )
```

Comment:
Can you not port my test addition from #557? A simple way to verify that this approach fixes the visible regression.

Reply:
Well given that your test addition is just a vanity test addition and does not actually test the issue reported (just very convincingly appears to) I'd rather add a correct set of tests for that issue.